packages/front-end/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,80 @@
+import { Flex } from "@radix-ui/themes";
+import clsx from "clsx";
+import { PiCheck, PiCircleDashed } from "react-icons/pi";
+
+type Step = {
+  label: string;
+  enabled: boolean;
+};
+
+interface Props {
+  step: number;
+  setStep: (step: number) => void;
+  steps: Step[];
+  setError: (error: string) => void;
+  skipped?: Set<number>;
+  validateSteps?: (step: number) => Promise<void>;
+}
+
+export default function Stepper({
+  step,
+  setStep,
+  steps,
+  skipped,
+  setError,
+  validateSteps,
+}: Props) {
+  return (
+    <Flex
+      as="div"
+      justify={"start"}
+      className={"nav nav-default paged-modal-default"}
+      mb="4"
+    >
+      {steps.map(({ label, enabled }, i) => {
+        return (
+          <Flex
+            as="div"
+            align={"center"}
```

Comment:
Super nit: are we able to make these just `"center"` instead of `{"center"}` there is a few places this is happening